### PR TITLE
New CSS Painting API feature

### PR DESCRIPTION
**Pull request by captainbrosset**

### CODE REVIEW

File: features/paint.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name: paint()
+description: The `paint()` CSS function paints an element's background or border using a custom paint worklet.
```
Comment (ddbeck) [inline features/paint.yml:2]: I didn't really know what "paint" meant here, so after reading the docs…

```suggestion
description: The `paint()` CSS function creates a custom image, drawn using a paint worklet, as in an element's `background-image` or `border-image` property.
```

I'm not sure that's a good description, but I think it adds a little more info. What do you think?

File: features/paint.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+spec: https://drafts.css-houdini.org/css-paint-api-1/
+caniuse: css-paint-api
+compat_features:
+  - css.types.image.paint
+  - api.CSS.paintWorklet_static
+  - api.PaintWorkletGlobalScope
+group: css
```
Comment (ddbeck) [inline features/paint.yml:9]: a real nit here (and I should write a script to format these automatically) but, if you're editing this file anyway, then moving this after `spec` will make this more consistent with other entries.

File: features/paint.yml (hunk 1 above)
Comment (captainbrosset) [reply features/paint.yml:2]: Thanks, feels better. I'm a bit confused by the "as in" part at the end. I'll think of it and push a new commit.

**Review by foolip — APPROVED** [review]

**Comment by foolip** [comment]
Thank you @captainbrosset!

**Pull request merged**